sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClient.cs

```diff
@@ -201,16 +204,17 @@ public virtual Response<string> GetDigitalTwin(string digitalTwinId, Cancellatio
         /// Console.WriteLine($&quot;Created digital twin &apos;{customDtId}&apos;.&quot;);
         /// </code>
         /// </example>
-        public virtual Task<Response<string>> CreateDigitalTwinAsync(string digitalTwinId, string digitalTwin, CancellationToken cancellationToken = default)
+        public virtual Task<Response<string>> CreateDigitalTwinAsync(string digitalTwinId, string digitalTwin, DigitalTwinsAddOptions options = null, CancellationToken cancellationToken = default)
         {
-            return _dtRestClient.AddAsync(digitalTwinId, digitalTwin, cancellationToken);
+            return _dtRestClient.AddAsync(digitalTwinId, digitalTwin, options, cancellationToken);
         }
 
         /// <summary>
         /// Creates a digital twin synchronously.
         /// </summary>
         /// <param name="digitalTwinId">The Id of the digital twin.</param>
         /// <param name="digitalTwin">The application/json digital twin to create.</param>
+        /// <param name="options">The optional parameters for this request.</param>
```

Comment:
You have `The optional settings for this request.` in some other places ... I think we should either be specific about the options or have a consistent comment for all of them.